### PR TITLE
Issue 244: make nav bar normal font-weight

### PR DESCRIPTION
**Pull request by shaftoes**
Made nav bar normal font-weight by removing css attribute in common.css

### CODE REVIEW

**Comment by david-yz-liu** [comment]
@shaftoes You found the right place in the CSS, but this isn't the change to make; you'll need to change cssGen.hs, which is the haskell file that generates all the CSS. My fault, I forgot to mention that.

**Comment by shaftoes** [comment]
Whoops! Do I create a new pull request with the actual changes?

**Comment by david-yz-liu** [comment]
It's fine, you can keep adding to this one. Your original changes will be overridden when you run cssGen.hs again.

**Comment by david-yz-liu** [comment]
@shaftoes You'll need to create a new selector along the lines of

```
#nav-links a {
    font-weight: normal
}
```

(but in cssGen.hs, of course)

**Comment by Ian-Stewart-Binks** [comment]
@shaftoes I may have missed something, why did you move `fontWeight bold` up to line 84?

**Comment by shaftoes** [comment]
@Ian-Stewart-Binks Nope, that was a typo -- I originally took it out and accidentally placed it two lines up. Sorry for all the mistakes, a lot of this is very new to me

**Comment by Ian-Stewart-Binks** [comment]
@shaftoes No worries! I've made plenty of mistakes myself.

File: hs/CssGen.hs [hunk 1]
```diff
@@ -77,7 +79,9 @@ headerCSS = do
                 a <? do
                     color white
 
+
```
Comment (david-yz-liu) [inline hs/CssGen.hs]: Remove these blank lines.

File: style/common/common.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
+
```
Comment (david-yz-liu) [inline style/common/common.css]: Remove this file from the commit. (You can do this by doing a "git rm style/common/common.css" and committing.)

**Comment by david-yz-liu** [comment]
The blank lines still need to be removed.

**Pull request merged**